backstop/tests/modal.html

```diff
@@ -40,15 +40,30 @@ <h1>Modal</h1>
 
     <a
       id="open-modal"
-      onclick="openModal()"
+      onclick="toggleModal(false)"
       class="iui-anchor"
     >Open modal</a>
 
+    <br>
+
+    <a
+      id="open-full-page-modal"
+      onclick="toggleModal(true)"
+      class="iui-anchor"
+    >Open full page modal</a>
+
     <script>
-      function openModal() {
+      function toggleModal(isFullPage) {
```

Comment:
Javascript function does not work well and has bugs. Needs to be fixed.

Reply:
The problem that is that when switching between the two types of modals, the beginning transitions are cut out if the opposite modal was just viewed (or if the full-page modal is the first viewed). So the full-page modal doesn't translate onto the page and the regular modal doesn't fade onto the page (I think, the fade is harder to tell). I think this happens because the "visibility" class name is toggled on before the modal has a chance to 'reset'? I tried to put a delay on the "toggle" code but that didn't seem to do anything.

Reply:
Maybe just have separate modals for these tests.

Reply:
I changed how the full page modal is implemented a little bit and ended up separating the modals.